### DISCUSSION TITLE
PDM always chooses `__pypackages__`

### DISCUSSION THREAD

**Discussion by adriangb** [pdm-project/pdm#2174] [Q&A] [answered]
Using PDM version 2.8.2 it seems to always choose `__pypackages__` over `.venv` even if I manually create the `.venv` then run PDM install. I've cleaned the repo (there is no `__pypackages__` folder) and wiped the PDM config as far as I can tell. Am I missing something? Is there any way to completely disable PEP 582 mode?

**Comment by frostming** [comment]
How to reproduce?

**Reply by adriangb** [reply]
That's the thing, I don't seem to be able to reproduce this, at least not in a clean Docker container. I've tried uninstalling PDM via `curl -sSL https://pdm.fming.dev/install-pdm.py | python3 -` / `curl -sSL https://pdm.fming.dev/install-pdm.py | python3 - --remove` but the issue persists. I tried deleting the configs in `~/Library/Application\ Support/pdm` (I'm on a Mac) but again no change. I get the feeling there's some misconfiguration somewhere but I don't know where else PDM could store configs or why it thinks it needs to use `__pypackage__` even when run from an activated venv. So any pointers in troubleshooting / cleanup would be appreciated. I'd also be happy just disabling anything PEP 582 related since I don't use it.

**Reply by frostming** [reply, marked as answer]
```
pdm config
pdm info
env |grep -i pdm
```

**Reply by adriangb** [reply]
Okay looks like `PDM_IGNORE_ACTIVE_VENV` was being set, I found it and deleted it, all good now. Appreciate the help.
